walkeeper/src/replication.rs

```diff
@@ -221,6 +248,27 @@ impl ReplicationConn {
         };
         info!("Start replication from {:?} till {:?}", start_pos, stop_pos);
 
+        // Don't spam pageserver with callmemaybe queries
+        // when connection is already established.
```

Comment:
slightly better to do that only when connection is indeed from pageserver, i.e. appname is not wal_proposer_recovery, see several lines above

Reply:
Fixed. Do we need any similar handling in `ReceiveWalConn::run()`?

Reply:
No, it can only receive WAL from the proposer.